applications/live-chat.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Real time machine translation of multilingual chat interactions
+---
+
+# Live Chat
+
+In **Live Chats**, machine translation assists in interactions between people speaking different languages. Each user sends messages in their own language, which get translated before reaching the other person. This means that both inbound and outbound messages are machine translated in real time.
```

Comment:
Machine translation for **live chat** is used to translate messages between two users who do not speak a common language.

Live chat messages are a type of [user-generated content](user-generated-content).  Live chat is challenging for machine-translation because it is noisy and context-dependent.

Common scenarios are in-game chat in [gaming](gaming) and customer support for [commerce and marketplaces](commerce-and-marketplaces).

### Companies
- Unbabel
- Language I/O